assets/js/llms-metabox-students.js

```diff
@@ -117,13 +123,27 @@
 			this.update_student_enrollment( $el.attr( 'data-id' ), 'remove' );
 		};
 
+		/**
+		 * Delete a Student's enrollment.
+		 *
+		 * @since 3.0.0
```

Comment:
remove duplicate `@since` in favor of `@since [version]` below

Reply:
right
fixed in 792ee4f